docs/user-manual.md

```diff
@@ -0,0 +1,80 @@
+
+The CPT tool is designed to be used with the data from CERN bubble chamber experiments that have been digitised and stored in the PartII Particle Tracks lab[^1]. The software allows the user to load and visualise the data, and to perform some basic analysis on the tracks. This manual describes how to use the tool to perform the measurements required for the practical.
+
+
+## Usage
+
+### Starting the tool
+
+To start the tool, double-click on the CPT icon. This will open the `napari` window. To load the CPT plugin, click on the `Plugins` menu and select `cavendish-particle-tracks`. This will open the CPT window, which will be empty until you load a data file.
+
+### Loading a data file
+To load a data file, click on the `Load data` button. This will open a file dialog, which will allow you to select the data folder you want to load. The data folder should have three subfolders corresponding to the three views of the detector. Each subfolder should contain the same number of images, one for each frame of the experiment.
+
+In the Particle Tracks lab, the data is stored in the `CPT_data` folder. Please note that the images in the subfolders might not correspond to the view indicated in the folder name. You will need to check the images to determine which view they correspond to.
+
+The data is loaded as a 4D array, with the dimensions corresponding to the views, the frames, the height and the width of the images. In practice, this means that once the data is loaded, the first view of the first frame will be displayed. The bottom sliders labelled `Event` and `Views` allow you to toggle between the different frames and different views for each frame, respectively.
+The tool will also display the number of views and frames available, as well as the current view and frame.
+
+### Adding a new particle
+Once an interesting process is identified in the image, you can record information about that process. To start, you need to add a new particle. To do this, click on the `Add particle` button, and select the process you want to record.
+This will create a new particle object in the particle list, which will be displayed as a new entry in the table [^2].
+
+### Measuring particle properties
+
+Once a particle is added, you can measure its properties. To do this, select the particle from the particle list, as this will enable the measurement that can be performed. The properties that can be measured are:
+
+- `Decay length`: The length of the particle path, between the origin and decay vertices.
+
+- `Radius of curvature`: The radius of the circle that best fits the particle path.
+
+- `Decay angles`: The angles between the particle path and its decay products. This measurement only makes sense for particles that decay into two visible products (i.e. $\Lambda^0 \rightarrow p  \pi^-$).
```

Comment:
```suggestion
- `Decay angles`: The angles between the particle path and its decay products. This measurement only makes sense for particles that decay into two visible products (i.e. {math}`\Lambda^0 \rightarrow p  \pi^-`).
```

We're using the `myst` markdown parser, [docs here](https://myst-parser.readthedocs.io/en/latest/syntax/math.html#math-and-equations)